spec/support/helpers/feature_spec_helpers.rb

```diff
@@ -156,4 +180,14 @@ def wait_for_trumbowyg(selector)
   def trumbowyg_ready?(selector)
     page.evaluate_script("$('#{selector}').trumbowyg('html') !== false")
   end
+
+  def wait_for_dropzone_upload
+    Timeout.timeout(Capybara.default_max_wait_time) do
+      loop until dropzone_ready?
+    end
+  end
+
+  def dropzone_ready?
+    page.evaluate_script("ELMO.mediaUploaderManager.is_uploading()")
+  end
```

Comment:
Lets put these dropzone things in a separate shared context and include it where necessary. No need to pollute everything. Can you also please do that for the trumbowyg ones. Thanks.